CRITICAL - 2024-07-20 01:12:04 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:12:10 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:14:21 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:14:23 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:14:25 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:15:16 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:15:22 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:15:27 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:17:37 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:17:38 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:17:39 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:17:39 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:17:39 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:17:40 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:17:40 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:17:40 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:17:47 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2024-07-20 01:20:46 --> ErrorException: Cannot redeclare App\Controllers\Ciudadanos::nuevoCiudadano()
[Method: GET, Route: ver_ciudadanos]
in APPPATH\Controllers\Ciudadanos.php on line 18.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
ERROR - 2024-07-20 01:39:43 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ci...', 0)
#1 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ci...')
#2 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ci...')
#3 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ci...', Array, false)
#4 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\app\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
#9 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
#10 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-20 01:39:43 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
[Method: POST, Route: agregar_ciudadano]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 01:39:43 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 01:39:43 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-20 01:41:16 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ci...', 0)
#1 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ci...')
#2 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ci...')
#3 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ci...', Array, false)
#4 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\app\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
#9 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
#10 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-20 01:41:16 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
[Method: POST, Route: agregar_ciudadano]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 01:41:16 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 01:41:16 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'120\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-20 01:41:19 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ci...', 0)
#1 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ci...')
#2 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ci...')
#3 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ci...', Array, false)
#4 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\app\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
#9 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
#10 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-20 01:41:19 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
[Method: POST, Route: agregar_ciudadano]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 01:41:19 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 01:41:19 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Ciudadanos.php(35): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-20 01:42:26 --> mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE) in C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ci...', 0)
#1 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ci...')
#2 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ci...')
#3 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ci...', Array, false)
#4 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\app\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
#9 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
#10 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-20 01:42:26 --> CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
[Method: POST, Route: agregar_ciudadano]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 01:42:26 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 01:42:26 --> [Caused by] mysqli_sql_exception: Cannot add or update a child row: a foreign key constraint fails (`fs2024_ciudadanos`.`ciudadanos`, CONSTRAINT `ciudadanos_ibfk_3` FOREIGN KEY (`cod_muni`) REFERENCES `municipios` (`cod_muni`) ON DELETE CASCADE ON UPDATE CASCADE)
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'545645654\', \'5465454\', \'adihsah@\', \'2014-11-02\', \'5\', \'\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 01:54:02 --> CodeIgniter\HTTP\Exceptions\HTTPException: The route for "ciudadanos" cannot be found.
[Method: GET, Route: eliminar_ciudadano/858585]
in SYSTEMPATH\HTTP\RedirectResponse.php on line 64.
 1 SYSTEMPATH\HTTP\RedirectResponse.php(64): CodeIgniter\HTTP\Exceptions\HTTPException::forInvalidRedirectRoute('ciudadanos')
 2 APPPATH\Controllers\Ciudadanos.php(42): CodeIgniter\HTTP\RedirectResponse->route(false)
 3 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->eliminarCiudadano('858585')
 4 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
 5 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 6 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
 7 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
 8 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-20 02:12:11 --> mysqli_sql_exception: Duplicate entry '1212121212' for key 'PRIMARY' in C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ci...', 0)
#1 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ci...')
#2 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ci...')
#3 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ci...', Array, false)
#4 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\app\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
#9 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
#10 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-20 02:12:11 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1212121212' for key 'PRIMARY'
[Method: POST, Route: agregar_ciudadano]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 02:12:11 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1212121212' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'\', \'\', \'\', \'\', \'Selecciones su Nivel Académico\', \'\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'\', \'\', \'\', \'\', \'Selecciones su Nivel Académico\', \'\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 02:12:11 --> [Caused by] mysqli_sql_exception: Duplicate entry '1212121212' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'\', \'\', \'\', \'\', \'Selecciones su Nivel Académico\', \'\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'\', \'\', \'\', \'\', \'Selecciones su Nivel Académico\', \'\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'\', \'\', \'\', \'\', \'Selecciones su Nivel Académico\', \'\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
ERROR - 2024-07-20 02:12:47 --> mysqli_sql_exception: Duplicate entry '1212121212' for key 'PRIMARY' in C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php:312
Stack trace:
#0 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ci...', 0)
#1 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ci...')
#2 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ci...')
#3 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ci...', Array, false)
#4 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\BaseModel.php(840): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Model.php(802): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\app\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
#9 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
#10 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(325): CodeIgniter\CodeIgniter->run()
#12 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\system\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
#13 C:\xampp\htdocs\fs2024\Ci-Ciudadanos\public\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
#14 {main}
CRITICAL - 2024-07-20 02:12:47 --> CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1212121212' for key 'PRIMARY'
[Method: POST, Route: agregar_ciudadano]
in SYSTEMPATH\Database\BaseConnection.php on line 688.
 1 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 2 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 3 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 4 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 5 APPPATH\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 7 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
 8 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
10 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
11 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 02:12:47 --> [Caused by] CodeIgniter\Database\Exceptions\DatabaseException: Duplicate entry '1212121212' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 317.
 1 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'556453453\', \'4354345\', \'ehwbhewbf\', \'2026-03-04\', \'6\', \'1909\')')
 2 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'556453453\', \'4354345\', \'ehwbhewbf\', \'2026-03-04\', \'6\', \'1909\')')
 3 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 4 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 5 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 6 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 7 APPPATH\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert([...])
 8 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
 9 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
10 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
11 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
12 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
13 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
CRITICAL - 2024-07-20 02:12:47 --> [Caused by] mysqli_sql_exception: Duplicate entry '1212121212' for key 'PRIMARY'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 312.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(312): mysqli->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'556453453\', \'4354345\', \'ehwbhewbf\', \'2026-03-04\', \'6\', \'1909\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(734): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'556453453\', \'4354345\', \'ehwbhewbf\', \'2026-03-04\', \'6\', \'1909\')')
 3 SYSTEMPATH\Database\BaseConnection.php(648): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (\'1212121212\', \'Diaz\', \'Genesis\', \'6asdsabds\', \'556453453\', \'4354345\', \'ehwbhewbf\', \'2026-03-04\', \'6\', \'1909\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2332): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ciudadanos` (`dpi`, `apellido`, `nombre`, `direccion`, `tel_casa`, `tel_movil`, `email`, `fechanac`, `cod_nivel_acad`, `cod_muni`) VALUES (:dpi:, :apellido:, :nombre:, :direccion:, :tel_casa:, :tel_movil:, :email:, :fechanac:, :cod_nivel_acad:, :cod_muni:)', [...], false)
 5 SYSTEMPATH\Model.php(394): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(840): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(802): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Ciudadanos.php(34): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(933): App\Controllers\Ciudadanos->agregarCiudadano()
10 SYSTEMPATH\CodeIgniter.php(509): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Ciudadanos))
11 SYSTEMPATH\CodeIgniter.php(355): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 SYSTEMPATH\Boot.php(325): CodeIgniter\CodeIgniter->run()
13 SYSTEMPATH\Boot.php(67): CodeIgniter\Boot::runCodeIgniter(Object(CodeIgniter\CodeIgniter))
14 FCPATH\index.php(56): CodeIgniter\Boot::bootWeb(Object(Config\Paths))
